java/src/jmri/jmrit/logix/OBlock.java

```diff
@@ -66,7 +66,7 @@ public enum OBlockStatus {
         private static final Map<String, OBlockStatus> map = new HashMap<>();
         private static final Map<String, OBlockStatus> reverseMap = new HashMap<>();
 
-        private OBlockStatus(int status, String name, String descr) {
+        OBlockStatus(int status, String name, String descr) {
```

Comment:
Why is this constructor changed from `private` to `package private`?
If static analysis warns about it that it isn't used, it's better to remove it.

Reply:
This was flagged in Sonar Rules as a redundant modifier.
As it's an Enum constructor there's no need for the private.
I had to check this as well when I saw it!
https://rules.sonarsource.com/java/RSPEC-2333/

Reply:
You are right! I missed that it was an enum.